woocommerce/woocommerce-android
# Fix budget slider responsiveness

Part of: https://github.com/woocommerce/woocommerce-android/issues/10574


### Description

These small changes fix several issues with the budget slider responsiveness. Issues are described in this PR: https://github.com/woocommerce/woocommerce-android/pull/10709#discussion_r1483263929

### Testing instructions

- Open Blaze campaign creation flow
- Open edit budget screen
- Play sliding back and forth with the budget slider. Ensure that the total budget amount is responsive, and it always displays a value that is multiple of the campaign's duration value. 

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/2663464/f43dd6ce-b56d-45e0-94d5-f9d0f9a3e805
